stdlib/public/runtime/Errors.cpp

```diff
@@ -320,7 +320,7 @@ reportNow(uint32_t flags, const char *message)
 #else
   write(STDERR_FILENO, message, strlen(message));
 #endif
-#if defined(__APPLE__)
+#if defined(__APPLE__) && SWIFT_STDLIB_HAS_DARWIN_ASL
```

Comment:
Any reason to not just use `SWIFT_STDLIB_HAS_DRAWIN_ASL`?  That is, I think that we should just drop the `defined(__APPLE__)`.

Reply:
`SWIFT_STDLIB_HAS_DARWIN_ASL` defaults to 1 - on all platforms. I think I chose that to avoid complicating the CMake logic (to pick a different default per platform)... Do you want me to change that?